basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -2902,6 +2903,117 @@ public static String stripOffQuery(String urlString)
 		return retval;
 	}
 
+	// Copy an LTI content Item from an old site into a new site
+	// assumes legit access to both the old and new site
+	public static Object copyLTIContent(Long contentKey, String siteId, String oldSiteId)
+	{
+		LTIService ltiService = (LTIService) ComponentManager.get("org.sakaiproject.lti.api.LTIService");
+		Map<String, Object> ltiContent = ltiService.getContentDao(contentKey, oldSiteId, true);
+		return copyLTIContent(ltiContent, siteId, oldSiteId);
+	}
+
+	// Copy an LTI content Item from an old site into a new site
```

Comment:
not the proper comment style for methods

Reply:
Agreed